androidApp/src/main/java/com/mbta/tid/mbta_app/android/onboarding/OnboardingScreenView.kt

```diff
@@ -191,25 +200,31 @@ fun OnboardingScreenView(
         }
         OnboardingScreen.StationAccessibility -> {
             OnboardingPieces.PageBox(painterResource(R.mipmap.onboarding_background_map)) {
-                if (textScale < 2f) {
-                    OnboardingImage(
-                        R.drawable.accessibility_icon_accessible,
-                        size = 192.dp,
-                        offsetY = haloOffset
-                    )
-                }
                 OnboardingContentColumn {
+                    if (textScale < 1.5f) {
+                        Row(horizontalArrangement = Arrangement.Center) {
+                            Image(
+                                painterResource(R.drawable.accessibility_icon_accessible),
+                                modifier = Modifier.size(192.dp).weight(1f),
+                                contentDescription = null
+                            )
+                        }
+                    }
```

Comment:
On both android & iOS, the default placing of the image was too low as part of the background image. Moving it to the foreground seemed to work well, and we still hide the image if the text gets too big to fit it comfortably.